x-pack/plugins/search_inference_endpoints/common/translations.ts

```diff
@@ -21,29 +21,23 @@ export const CANCEL = i18n.translate('xpack.searchInferenceEndpoints.cancel', {
 export const MANAGE_INFERENCE_ENDPOINTS_LABEL = i18n.translate(
   'xpack.searchInferenceEndpoints.allInferenceEndpoints.description',
   {
-    defaultMessage: 'Manage your inference endpoints.',
-  }
-);
-
-export const ADD_ENDPOINT_LABEL = i18n.translate(
-  'xpack.searchInferenceEndpoints.newInferenceEndpointButtonLabel',
-  {
-    defaultMessage: 'Add endpoint',
+    defaultMessage:
+      'Inference endpoints streamline the deployment and management of machine learning models in Elasticsearch. Set up tasks such as text embedding, completions, and reranking using simple, unique endpoints.',
```

Comment:
```suggestion
      'View and manage your deployed inference endpoints.',
```

Reply:
@leemthompo @szabosteve - Some of the feedback we got was that "no one will understand what inference endpoints are." I thought y'all had a more text-heavy idea on this one?

Reply:
Updated based on @szabosteve's [recommendation](https://github.com/elastic/kibana/pull/188783/commits/be58fed3033d2cc5c79aefee4fb9ae150affb7e6).

Reply:
@daveyholler right as Saikat says we used the text-heavy explanation for the empty state